resources/event-bus/charts/subscription-controller-knative/templates/service-account.yaml

```diff
@@ -96,3 +96,27 @@ roleRef:
   kind: ClusterRole
   name: {{ .Release.Name }}-knative-subscriptions-controller
   apiGroup: rbac.authorization.k8s.io
+---
+### Events #####################################################
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ .Release.Name }}-events-controller
+rules:
+  - apiGroups: [""]
+    resources: ["events"]
+    verbs: ["get", "watch", "list", "create", "update", "patch", "delete"]
```

Comment:
perhaps "create", "update", "patch" will be enough

Reply:
Done